ga4gh/datamodel/variants.py

```diff
@@ -334,6 +334,195 @@ def generateVariant(self, referenceName, position, randomNumberGenerator):
         return variant
 
 
+class SimulatedVariantAnnotationSet(AbstractVariantSet):
+    """
+    A variant annotation set that doesn't derive from a data store.
+    Used mostly for testing.
+    """
+    def __init__(
+            self, parentContainer, localId, variantSet, randomSeed=1):
+        super(SimulatedVariantAnnotationSet, self).__init__(
+            parentContainer, localId)
+        self._variantSet = variantSet
+        self._variantSetId = str(variantSet.getCompoundId())
+        self._randomSeed = randomSeed
+        now = protocol.convertDatetime(datetime.datetime.now())
+        self._compoundId = datamodel.VariantAnnotationSetCompoundId(
+            self.getCompoundId(), 'variantannotations')
+        self._creationTime = now
+        self._updatedTime = now
+        # TODO make some reasonable connection to seqont
+
+    def getAnalysis(self):
+        analysis = protocol.Analysis()
+        analysis.recordCreateTime = datetime.datetime.now().strftime(
+                                    "%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
This is definitely going to cause problems with later versions of flake8. Hanging indents indent by 4 spaces.